Gemfile

```diff
@@ -4,6 +4,8 @@ source "https://rubygems.org"
 gem "jekyll", "~> 4.3"
 gem "webrick", "~> 1.7"
 
+gem "html-proofer"
```

Comment:
Seeing the pattern above with version specifications, would it make sense to continue that pattern here for `html-proofer`?